README.md

```diff
@@ -3,6 +3,24 @@
 [Hyperledger Besu](https://github.com/hyperledger/besu/) is a Hyperledger enterprise-grade Java-based,
 Apache 2.0 licensed Ethereum client.
 
+Welcome to the **Besu Documentation** repository.
 The software sources are hosted in https://github.com/hyperledger/besu
 
 This repository only contains the sources for [Besu documentation website hosted by ReadTheDocs](https://besu.hyperledger.org/).
+
+
+## Useful Links
+
+* [Besu Docs](http://besu.hyperledger.org/)
+* [Besu Jira](https://jira.hyperledger.org/projects/BESU/issues)
+* [Besu Wiki](https://wiki.hyperledger.org/display/BESU/Hyperledger+Besu)
+* [How to Contribute to the Documentation](CONTRIBUTING.md)
+* How to Contribute to the [Besu Codebase](https://github.com/hyperledger/besu/blob/master/CONTRIBUTING.md)
+
+
+## LF / Jira Accounts
+
+Having the following accounts is necessary for contributing code/issues to Besu.  
+* If you want to contribute code, you can make a [LF foundation account here](#).  
```

Comment:
The F in LF is already for foundation ;)
Maybe always use `LF` or always `Linux Foundation (LF)` I don't know, but something user can understand simply (LF is not so easy when you don't know much about the foundations around Besu, HLF, LF... confusing) so maybe use the full word with acronym between parentheses.

Reply:
And link is just # it's not going to work.